toolsrc/include/vcpkg/base/system.process.h

```diff
@@ -23,6 +23,18 @@ namespace vcpkg::System
                                      const fs::path& cmake_script,
                                      const std::vector<CMakeVariable>& pass_variables);
 
+    struct CmdLineBuilder
+    {
+        CmdLineBuilder& path_arg(const fs::path& p) { return string_arg(p.u8string()); }
+        CmdLineBuilder& string_arg(StringView s);
+        std::string build() { return std::move(buf); }
```

Comment:
Don't like this `build()` function silently being a move. How about:

```suggestion
        const std::string& build() const& noexcept { return buf; }
        std::string&& build() && noexcept { return std::move(buf); }
```